great_expectations/expectations/registry.py

```diff
@@ -52,6 +52,11 @@
 """
 
 
+class RendererImpl(NamedTuple):
+    expectation: str
+    renderer: Callable
```

Comment:
Is there anything else we can say about the interface of the `renderer` `Callable`, like return type, number of arguments etc?
https://mypy.readthedocs.io/en/stable/kinds_of_types.html#callable-types-and-lambdas

Reply:
Great question, I updated this annotation to be more specific!